.travis.yml

```diff
@@ -9,27 +9,34 @@ services:
 env:
   global:
     - REMOTE_IMAGE="ezsystems/php"
-    - LATEST="7.2"
-    - FORMAT_VERSION="v1"
+    - LATEST_PHP="7.3"
+    - LATEST_NODE="12"
+    - FORMAT_VERSION="v2"
   matrix:
-    # Run once per Dockerfile-<PHP_VERSION>
-    - PHP_VERSION="5.6" XDEBUG_CHANNEL="xdebug-2.5.5" EZ_VERSION="^1.13.4"
-    - PHP_VERSION="7.0" XDEBUG_CHANNEL="xdebug-2.8.1" EZ_VERSION="^1.13.4"
-    - PHP_VERSION="7.1"
-    - PHP_VERSION="7.2"
-    - PHP_VERSION="7.3"
+    # Run per Dockerfile-<PHP_VERSION>-<NODE_VERSION>
+    - PHP_VERSION="7.1" NODE_VERSION="10" EZ_VERSION="^2.5"
```

Comment:
Do we need v10 for anything going forward, or could we drop it on v2?


Besides that, looks very promising :)

Reply:
I wanted to keep Node10, because we don't bump Node requirement for v2: https://github.com/ezsystems/developer-documentation/pull/1012#discussion_r404819609 

So it might be useful for testing sometimes.

Reply:
Ok, fair enough.

My thinking was given v10 has end of life in under a year we _should_ anyway update doc to reflect all (newer) node.js versions we technically support both for v2.5 and 3.0, _if_ our dependencies now actually work on newer versions.